validator/errors/errors.py

```diff
@@ -0,0 +1,22 @@
+"""Custom Errors for Nestor Config Validator"""
+
+
+class InvalidTargetPathError(Exception):
+    """Exception raised when the target path (NESTOR_TARGET_PATH) env is not set"""
```

Comment:
Just a thought but what about having the exception raised directly from `get_target_path()` ?
It would solve the duplicate you have with `build_app_path`/`build_project_conf_path` being too similar.

I don't know if there are cases where the variable being missing can be acceptable 🤷